src/test/java/com/appaloosastore/client/AppaloosaClientTest.java

```diff
@@ -484,16 +500,36 @@ public void constructParametersFromUpdateShouldUseGroupsIfSpecified()
 	public void setUpdateParametersShouldUpdateDescriptionAndGroupNames(){
 		MobileApplicationUpdate update = new MobileApplicationUpdate();
 		String description = "New Desc";
+		String changelog = null;
 		List<String> groupNames = new ArrayList<String>();
 		groupNames.add("Group1");
 		groupNames.add("Group2");
 		
-		appaloosaClient.setUpdateParameters(update, description, groupNames);
+		appaloosaClient.setUpdateParameters(update, description, groupNames, changelog);
 		
 		assertEquals(description, update.description);
 		assertEquals(2, update.groupNames.size());
 		assertTrue(update.groupNames.contains("Group1"));
 		assertTrue(update.groupNames.contains("Group2"));
+		assertEquals(changelog, update.changelog);
```

Comment:
Also, test against a constant, so that we know that the called method does not perform some trickery with the object props